File: C & C++ PRACTICE CODES/INVERTED TRIANGLE PATTERN/code5.c
#include <stdio.h>


void main(){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int j = 0 ; j < row - i ; j++){

		

			if(row%2 == 1){

				if(j%2 == 0){

					printf("%d\t",j+1+i);
				}else{
					printf("%c\t",64+j+i+1);

				}
			}else{

				if(j%2 == 1){

					printf("%d\t",j+1+i);
				}else{
					printf("%c\t",64+j+i+1);

				}

			}


		}

		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/12/code5.java


import java.util.Scanner;

class PyramidDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter rows : ");
		int row = s.nextInt();
		int temp = 0;

		for(int i = 0 ; i < row ; i++){

			for(int k = 0 ; k  < row-i ; k++){

				System.out.print("		");
			}
			for(int  j = 0 ; j < 2*(i)+1; j++){

				if(i % 2 == 0){

					System.out.print((char)(65+j)+"		");
				}else{

					System.out.print((char)(97+j)+"		");
				}


			}

				System.out.print("\n\n");

		}

	}

}
